hack/update-k8sio-gomod-replace.sh

```diff
@@ -10,12 +10,27 @@ MODS=($(
     curl -sS https://raw.githubusercontent.com/kubernetes/kubernetes/v${VERSION}/go.mod |
     sed -n 's|.*k8s.io/\(.*\) => ./staging/src/k8s.io/.*|k8s.io/\1|p'
 ))
-for MOD in "${MODS[@]}"; do
-    V=$(
+
+# Export variables for access in subshells run by xargs
+export VERSION
+
+# Function to update go.mod replace directive for a module
+update_mod() {
+    local MOD=$1
+    local V=$(
         go mod download -json "${MOD}@kubernetes-${VERSION}" |
         sed -n 's|.*"Version": "\(.*\)".*|\1|p'
     )
     echo "Updating go.mod replace directive for ${MOD}"
     go mod edit "-replace=${MOD}=${MOD}@${V}"
```

Comment:
Wouldn't calling `go mod edit` in parallel cause unwanted artifacts in `go.mod` and/or `go.sum` 🤔 ? Cursory google search doesn't yield any results on this.

Reply:
Good point. The `go mod replace` commands are quick enough so we can safely parallelize the `go mod download`s only, still getting similar gains: 5d1c151